perllib/FixMyStreet/App/Controller/Report/New.pm

```diff
@@ -1671,9 +1671,7 @@ sub check_for_category : Private {
     my ( $self, $c, $opts ) = @_;
 
     my $category;
-    if ($c->get_param('category.NationalHighways') && $c->get_param('category.NationalHighways') !~ /\(NH\)/) {
-        $category = $c->get_param('category.NationalHighways');
-    } elsif (!$opts->{with_group}) {
+    if (!$opts->{with_group}) {
```

Comment:
I did not see what this code was for - I *think* it was because the client-side code ended up incorrectly deselecting the top level "National Highways" item which would then confuse both client and server code. That being fixed, I don't think any change is needed here.